lib/stripe/subscriptions/usage.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Stripe.SubscriptionItem.Usage do
+  @moduledoc """
+  Work with Stripe usage record objects.
+
+  Stripe API reference: https://stripe.com/docs/api/usage_records
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type t :: %__MODULE__{
+          id: Stripe.id(),
+          object: String.t(),
+          livemode: boolean,
+          quantity: non_neg_integer,
+          subscription_item: Stripe.id() | Stripe.SubscriptionItem.t() | nil,
+          timestamp: Stripe.timestamp() | nil
```

Comment:
Are timestamp and subscription item nullable?

I look in here usually: https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.yaml

and search `usage_record:`